src/test/java/fi/metatavu/keycloak/GraphApiTests.java

```diff
@@ -0,0 +1,93 @@
+package fi.metatavu.keycloak;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+
+public class GraphApiTests extends AbstractSeleniumTest {
+
+    private static final Network network = Network.newNetwork();
+
+    static GenericContainer keycloakContainer;
+
+    static GenericContainer wiremockContainer;
+
+    static BrowserWebDriverContainer webDriverContainer;
+
+    @BeforeAll
+    public static void setup() throws IOException, InterruptedException {
+        keycloakContainer = new GenericContainer(DockerImageName.parse("quay.io/keycloak/keycloak:26.1.2"));
+        keycloakContainer.withNetwork(network);
+        keycloakContainer.addFileSystemBind("./build/libs/", "/opt/keycloak/providers", BindMode.READ_ONLY);
+        keycloakContainer.addFileSystemBind("./src/test/resources/kc.json", "/opt/keycloak/data/import/kc.json", BindMode.READ_ONLY);
+        keycloakContainer.withNetworkAliases("keycloak");
+
+        keycloakContainer.withExposedPorts(8080);
+        keycloakContainer.withEnv("GRAPH_API_URL", "http://wiremock:8080");
+        keycloakContainer.withCommand("start-dev", "--import-realm");
+        keycloakContainer.start();
+
+        wiremockContainer = new GenericContainer(DockerImageName.parse("wiremock/wiremock:3.3.1"));
```

Comment:
Is there a reason to not use https://github.com/wiremock/wiremock-testcontainers-java?